examples/demo_dlmbl/solution.py

```diff
@@ -1,53 +1,81 @@
 # %% [markdown]
 """
-# Image translation
----
+<div style="text-align: left;">
```

Comment:
Why does this whole section need to be HTML?

Reply:
It might have been a VSCode Jupyter thing, but I was not able to render it properly if it wasn't HTML.